settings/common.py

```diff
@@ -297,3 +298,9 @@
 REDOC_SETTINGS = {
     'SPEC_URL': ('docs.yaml', {'format': '.yaml'}),
 }
+
+# Datetime warning settings
```

Comment:
Also, instead of `# Datetime warning settings` please add briefly why is the below code added.